demo/src/main/java/org/gbif/pipelines/demo/utils/ArgsParser.java

```diff
@@ -0,0 +1,31 @@
+package org.gbif.pipelines.demo.utils;
+
+/**
+ * Parser for command line arguments.
+ */
+public final class ArgsParser {
```

Comment:
we should remove this class and use custom options as described here: https://beam.apache.org/documentation/programming-guide/#creating-a-pipeline